addons/base_address_extended/migrations/13.0.1.0/openupgrade_analysis.txt

```diff
@@ -1,4 +1,4 @@
 ---Models in module 'base_address_extended'---
 ---Fields in module 'base_address_extended'---
 ---XML records in module 'base_address_extended'---
----nothing has changed in this module--
+NEW res.country: base.de (noupdate)
```

Comment:
?

Reply:
https://github.com/odoo/odoo/commit/2b674478ad50ca6c6caffb9a6efc69e265571508

Reply:
Yeah, but the question is that this shouldn't be recognized as a change, as the module is not the same

Reply:
The entry in this analysis means that a new record in the module with xmlid "base.de" is created (or modified in case it already exists in a module dependency). And it has always been this way. If you want it to be different, then propose how would you like to be the entry and then I may study how to implement it.

Reply:
Is this detected by compare noupdate script? If so, then I propose to change the label `ADD` to something like `UPD` or similar.

Reply:
Well, I will create an issue, and in the future try to do it.

Reply:
https://github.com/OCA/OpenUpgrade/issues/2412